jacksonrudderadapter/gradle-updater.js

```diff
@@ -0,0 +1,22 @@
+var PropertiesReader = require('properties-reader');
+module.exports.readVersion = function (contents) {
```

Comment:
if this file is always the same we can move it to the root of monorepo and reffer to it from all versionrc files

Reply:
The properties file are in individual folders. Can not access them individually if it's moved out